Resources/doc/config.rst

```diff
@@ -405,9 +405,10 @@ Otherwise you will get a *auth failed* exception.
             connections:
                 default:
                     server: "mongodb://localhost:27017"
-                    username: someuser
-                    password: somepass
-                    authSource: db_you_have_access_to
+                    options:
+                        username: someuser
+                        password: somepass
+                        db: db_you_have_access_to
```

Comment:
We're having an old problem: the documentation is generated from the master branch, which isn't released yet. I'd assume you're using a stable (3.2) release which doesn't yet have the `authSource` option. We could add a note about the `authSource` option being introduced in an upcoming release.